### DISCUSSION TITLE
我该怎么给formily加边框

### DISCUSSION THREAD

**Discussion by zhoupeng007** [alibaba/formily#3683] [Q&A 2.x]
有大佬知道我该怎么把 formily 写成这样啊  formily支持嘛还是我需要样式覆盖   
![DDD8C5BC-6838-412F-BADC-ADA4BC61B438](https://user-images.githubusercontent.com/49739368/215255193-ae2fb454-0e84-477c-b0ae-dc230a5b396e.png)

**Comment by yanhao98** [comment]
我认为：你需要自己做组件桥接。

**Reply by zhoupeng007** [reply]
有相关的文档吗~

**Reply by yanhao98** [reply]
参照源码自己写组件桥接。我是这样做的。

**Reply by zhoupeng007** [reply]
能给出一个demo嘛~多谢~

**Reply by yanhao98** [reply]
<img width="1907" alt="image" src="https://user-images.githubusercontent.com/37316281/215463539-af8fa2df-2234-4725-bbb8-a92845a7e8cd.png">

大概这样，然后参考源码。就可以做到完全自定义了。
